sourcegraph/sourcegraph
# Remove "Post Sign Up" / "Teams beta" step

Closes https://github.com/sourcegraph/sourcegraph/issues/33309.
## Test plan
- `sg start dotcom`
- Sign in and open https://sourcegraph.test:3443/welcome?debug=1
- Check that flow doesn't contain "Teams beta" step


## Screenshots
| BEFORE | AFTER |
| --: | :-- |
| <img width="880" alt="image" src="https://user-images.githubusercontent.com/6717049/163579269-ce96ba10-e163-47b8-bf25-ae54a635ac06.png"> |  <img width="880" alt="image" src="https://user-images.githubusercontent.com/6717049/163581067-bbc2d092-5d57-4234-8e8e-c3cfc760b7a8.png"> |
